### PR TITLE
add zh /docs/reference/glossary/limitrange.md

### PR DESCRIPTION
**Pull request by lichuqiang**
to address https://github.com/k8smeetup/website-tasks/issues/3195

### CODE REVIEW

File: content/zh/docs/reference/glossary/limitrange.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+ Provides constraints to limit resource consumption per {{< glossary_tooltip text="Containers" term_id="container" >}} or {{< glossary_tooltip text="Pods" term_id="pod" >}} in a namespace.
+-->
+ 提供约束来限制命名空间中每个 {{< glossary_tooltip text="容器" term_id="container" >}} 或 {{< glossary_tooltip text="Pod" term_id="pod" >}} 的资源消耗。
+
+<!--more--> 
+<!--
+LimitRange limits the quantity of objects that can be created  by type, 
+as well as the amount of compute resources that may be requested/consumed by individual {{< glossary_tooltip text="Containers" term_id="container" >}} or {{< glossary_tooltip text="Pods" term_id="pod" >}} in a namespace.
+-->
+LimitRange 按照类型来限制命名空间中对象能够创建的数量，以及单个 {{< glossary_tooltip text="容器" term_id="container" >}} 或 {{< glossary_tooltip text="Pod" term_id="pod" >}} 可以请求/使用的计算资源量。
```
Comment (tengqm) [inline content/zh/docs/reference/glossary/limitrange.md]:
```suggestion
LimitRange 用来限制某名字空间中可基于类型创建的对象的数量，各个 {{< glossary_tooltip text="容器" term_id="container" >}} 或 {{< glossary_tooltip text="Pod" term_id="pod" >}} 可以请求/使用的计算资源量。
```

Comment (lichuqiang) [reply content/zh/docs/reference/glossary/limitrange.md]: can't agree.

按照类型来限制是前提，放到后面来表示读起来很奇怪

Comment (tengqm) [reply content/zh/docs/reference/glossary/limitrange.md]: > LimitRange limits the quantity of objects that can be created  by type

这句话确实读起来有歧义。可以认为 "can be created by type" 是用来修饰 objects；也可以认为 "by type" 是用来补足 limits 这个动词。

Comment (markthink) [reply content/zh/docs/reference/glossary/limitrange.md]: 我认为差异不大..

Comment (tengqm) [reply content/zh/docs/reference/glossary/limitrange.md]: Revisited the API spec: https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.16/#limitrangeitem-v1-core

The current translation is acceptable. LimitRange is done by type.

**Pull request merged**